earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -290,12 +290,20 @@ ()
 {
     // make a copy since actualinputarg is macro substituted, so begin() &
     // end() could be called on different temporary objects
-    std::vector<std::string> inputarg = $actualinputarg;
+    MTVector<SharedString> inputarg = $actualinputarg;
     // bind my input versions refs
-    std::vector<std::string> boundInputs;
-    boundInputs.reserve(inputarg.size());
-    std::transform(inputarg.begin(), inputarg.end(), back_inserter(boundInputs),
-                   ptr_fun(&AssetVersionRef::Bind));
+    // Make a copy using the copy constructor
+    MTVector<SharedString> boundInputs(inputarg);
+
+//TODO - double check that this works
```

Comment:
Minor: could remove these comments, and the ones a few lines below, if they're not needed.